src/content/en/2019/performance.md

```diff
@@ -34,7 +37,7 @@ Experiences are measured monthly including paint, load, and interactivity metric
 
 So let's dive in and see what insights we can find.
 
-### FCP
+### First Contentful Paint (FCP)
```

Comment:
Not sure I agree with expanding these abbreviations. They're all defined earlier in the document. So now they're defined in both "The state of performance" and each heading. Is this a best practice that I wasn't aware of?

Reply:
Well as someone less familiar with at least some of these acronyms I struggled to understand them when trying to pull out the key details for the Meta Descriptions. The Table of Contents for this chapter isn't the friendliest with so many abbreviations and I think first use of them in ToC should be expanded - in the same way that first use of them in the text is.

Reply:
Going to have a second go at persuading you on this one for the following reasons:

- [Our style guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Wiki) emphasis accessibility, writing for global audience and writing inclusive documentation. I think a ToC full of acronyms goes against that spirit.
- The Chapter ToC is the first text you see - before the explanations embedded in the text.
- There is precedence is [avoid abbreviations in headings in other style guides](https://blog.apastyle.org/apastyle/abbreviations/#Q8), though I think avoiding them completely in headings and writing them all out would be a step too far given how many you've used here.
- Some of the terms (e.g. TTFB) are familiar in web performance circles but others are newish (e.g. FCP) and some (e.g. ECT) would be less well known (I didn't know this last one and had to hunt for it!)

Persuaded you yet? 😀

Reply:
Ok ok let's do it your way 😁 

But for conciseness could we remove the acronym from the parenthesis? So just `First Contentful Paint` as the heading rather than `First Contentful Paint (FCP)`.

Reply:
Done!